src/test/java/core/basesyntax/service/implementation/DataConversionServiceImplTest.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax.service.implementation;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FileReaderService;
+import java.util.List;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class DataConversionServiceImplTest {
+    private DataConversionServiceImpl dataConversionService;
+    private FileReaderService fileReaderService;
+    private String newFile;
+    private String fileLines;
+
+    @BeforeEach
+    void setUp() {
+        dataConversionService = new DataConversionServiceImpl();
+        fileReaderService = new FileReaderServiceImpl();
```

Comment:
do not create services before each test

Reply:
Also all your test has to be isolated, so do not use fileReader service in dataConversionTest. Just make hardcoded data to convert

Reply:
fix it for all your tests